BookingSystem/BookingDatabase/Bookings.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Bookings]
+(
+	[BookingId] INT IDENTITY(1,1) NOT NULL PRIMARY KEY,
+	[CreatedDate] DATETIME2 NOT NULL,
+	[MoveInDate] DATETIME2,
+	[MoveOutDate] DATETIME2,
+	[TotalPrice] DECIMAL(18, 4),
+	[Status] TINYINT NOT NULL,
+	[RoomNumberId] INT NOT NULL,
+	[UserId] INT NOT NULL,
+    CONSTRAINT [FK_BookingRoomNumber] FOREIGN KEY ([RoomNumberId]) REFERENCES [RoomNumbers] ([RoomNumberId]),
+	CONSTRAINT [FK_BookingUser] FOREIGN KEY ([UserId]) REFERENCES [Users] ([UserId]),
```

Comment:
You can separate table names with "_" for better readability (FK_Booking_User). This is just a recommendation